x-pack/test/api_integration/apis/kibana/kql_telemetry/kql_telemetry.js

```diff
@@ -10,12 +10,8 @@ import expect from '@kbn/expect';
 export default function ({ getService }) {
   const supertestNoAuth = getService('supertestWithoutAuth');
   const supertest = getService('supertest');
-  const esArchiver = getService('esArchiver');
 
   describe('telemetry API', () => {
-    before(() => esArchiver.load('empty_kibana'));
```

Comment:
Context on this change?  Is empty_kibana implied?

Reply:
"resetting" the Kibana index wasn't necessary. It must have been at one point. If it's still needed we can use spaces for isolation, and/or a Kibana export archive to achieve some default objects.

Reply:
With as much "thrashing" of the `.kibana` index as we have been doing, this might have been needed to even get it to a good place previously. I have a follow-up PR to reduce this even further.